ptsprojects/zephyr/iutctl.py

```diff
@@ -183,11 +186,11 @@ def reset(self):
         """Restart IUT related processes and reset the IUT"""
         log("%s.%s", self.__class__, self.reset.__name__)
 
```

Comment:
Why not just check if self.board is defined and if not then return?

Reply:
For qemu and native, we must restart the process to get the ready event!

Reply:
But `start()` is called before wait_iut_ready_event, so it is already started.